source/install/install-rhel-8.rst

```diff
@@ -98,4 +98,37 @@ If you wish to remove the Mattermost Server for any reason, you must stop the Ma
 
 	Depending on your configuration, there are several important folders in ``/opt/mattermost`` to backup. These are ``config``, ``logs``, ``plugins``, ``client/plugins``, and ``data``. We strongly recommend you back up these locations before running the ``rm`` command.
 
-You may also remove the Mattermost systemd unit file and the user/group created for running the application.
\ No newline at end of file
+You may also remove the Mattermost systemd unit file and the user/group created for running the application.
+
+Frequently asked questions
+--------------------------
+
+Can I run Mattermost without a proxy?
```

Comment:
I would place this after the FAQs about starting at boot and any mentions of nginx.